koku/api/dump/models.py

```diff
@@ -0,0 +1,48 @@
+"""Models for user-initiated data dump requests."""
+
+from uuid import uuid4
+
+from django.db import models
+
+
+class DumpRequest(models.Model):
+    """A user's request for a data dump."""
+
+    PENDING = 'pending'
+    COMPLETE = 'complete'
+    STATUS_CHOICES = ((PENDING, 'Pending'), (COMPLETE, 'Complete'))
+
+    uuid = models.UUIDField(default=uuid4, editable=False, unique=True, null=False)
+    created_by = models.ForeignKey('User', null=False, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
Just for consistency with other database models let's use `created_timestamp` and `updated_timestamp`.

Reply:
Updated to match in the latest push.